src/ControlLabel/ControlLabel.module.scss

```
@@ -0,0 +1,6 @@
+@import "../constants/mixins";
+
+.label {
+  @include fontStyles(body, 14, true);
```

can this just be bold?

```suggestion
  font-weight: blod
```